code/datums/mutations/body.dm

```diff
@@ -366,6 +372,12 @@
 /datum/mutation/human/gigantism/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
+	// SKYRAT EDIT BEGIN
+	if(owner.dna.features["body_size"] > 1)
+		to_chat(owner, "You feel your body expanding even further, but it feels like your bones are expanding too much! Uh oh!")
+		owner.gib(FALSE, FALSE, TRUE) // "lmao 200% big lemme just grab gigantism too"
+		return
```

Comment:
Read above comment. You arent gibbing people over this; ESPECIALLY because you arent always sure its intentional, such as what someone said, with their sprite being literally 5% bigger.